proposals/incubation/credebl.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: CREDEBL
+parent: Incubation
+grand_parent: Project Proposals
+---
+
+# CREDEBL Project Proposal
+
+## CREDEBL
+**CREDEBL** is a Decentralized Identity & Digital Credentialing Platform. It is recognized as a Digital Public Good (DPG) by the Digital Public Good Alliance (DPGA).
+
+# Sponsor(s)
+* _Ajay Jadhav (ajay@ayanworks.com_)
+* _Kalyan Kulkarni (kalyan@ayanworks.com)_
+* _Deepak Ghadge (deepak.ghadge@ayanworks.com)_
+* _Ankita Patidar (ankita.patidar@ayanworks.com)_
+* _Amit Padmani (amit.padmani@ayanworks.com)_
+* _Prasad Kalamkar (prasad.kalamkar@ayanworks.com)_
+
+# Abstract
+CREDEBL is an industry-leading Verifiable Credential & Decentralised Identity management platform which helps in building scalable, secure and user-centric SSI solutions. The multi-tenant platform offers straightforward interactions, enabling rapid configuration & deployment of any DID/VC solution.
```

Comment:
Is the code that will be contributed to LF Decentralized Trust intended to be complete such that someone else could run a multi-tenant platform? Or are there only pieces of the code that you intend to contribute such that the multi-tenant platform is something that will be an offering from AYANWORKS?

Reply:
The existing, complete multi-tenant capability will be part of the open-source contribution.